### DISCUSSION TITLE
ZepetoScript에서 UIBehaviour나, MaskableGraphic을 상속받고 싶습니다.

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#1245] [5. Q&A]
안녕하세요.

UI관련된 컴포넌트를 만들기 위해, UIBehaviour나, MaskableGraphic을 상속 받는 컴포넌트를 TS로 만들어서 사용하고 싶습니다.
(정확히는 MaskableGraphic만 상속 받을수 있어도 될것같네요.)

해당 컴포넌트를 상속받을수 있는 방법이 있으면 가이드 부탁드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 아쉽게도 현재는 지원하지 않는 기능입니다.
UIBehaviour 및 MaskableGraphic를 지원할 수 있도록 준비 중에 있으나, 반영 시기는 확답 드리기가 어려운 점 양해 부탁 드립니다.
감사합니다.
